README.md

```diff
@@ -1 +1,4 @@
-# hello-world
\ No newline at end of file
+# hello-world
+
+### Purpose
```

Comment:
Since there are no other sections present beyond the title, shouldn't this use H2 instead of H3?